### DISCUSSION TITLE
Change Library Folder Image For Webradio

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1287] [Fixed Issues] [answered]
@rern
I would like to choose a custom image for folders in the webradio library, just as we can for the streams. Is that a possibility?

**Comment by AndyFx12** [comment, marked as answer]
Hi R101G,
it´s possible yet - 2 options:
- bookmark your webradio-folder and change bookmark-coverart. Folder get´s same image at library list. Even after deleting the bookmark
- or copy image to "thumbs.jpg" in your webradio-folder
Andy

**Reply by R101G** [reply]
Thanks @AndyFx12. That works well.
